libs/front/components/tsconfig.json

```diff
@@ -3,6 +3,7 @@
   "compilerOptions": {
     "jsx": "react-jsx",
     "allowJs": true,
+    "resolveJsonModule": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "forceConsistentCasingInFileNames": true,
```

Comment:
After doing some research on issues in `monorepo` concerning TS husky pre-commit check (cf [here](https://hubloteam.slack.com/archives/C01CT4U1NKT/p1648562194527849)), I discovered we should [use some settings](https://www.typescriptlang.org/docs/handbook/project-references.html#composite) when dealing with `"references"`.